### PR TITLE
adds tabs to the tag content pane

### PR DESCRIPTION
**Pull request by jackcarlisle**
## ready for review

### CODE REVIEW

**Comment by nelsonic** [comment]
Codeship project: https://codeship.com/projects/142303

File: src/components/filter-buttons/index.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+import './styles.css';
+
+class FilterButtons extends Component {
+  render () {
+    return (
+      <ButtonGroup className='filterButtons'>
+      {filterButtonData.map(button => {
+        const selected = this.props.tagType === button.tagType ? 'primary' : 'default';
+        return (
+          <Button key={button.icon} bsStyle={selected} onClick={() => this.props.onFilterButtonClick(button.queryType, button.tagType)}>
```
Comment (nikhilaravi) [inline src/components/filter-buttons/index.js]: maybe split the Button props onto different lines so it is more readable.

File: src/components/linked-tags-list/index.js [hunk 2]
```diff
@@ -6,17 +6,18 @@ import './styles.css';
...
       selectedTagId
     } = this.props;
-    const list = listItems.map(item => {
+    const list = items.map(item => {
       return (
-        <div key={item._id} className='tag__item'>
-          <Tag key={item._id} id={item._id} tagName={item.displayName} selected={item._id === selectedTagId} onClick={handleTagClick} />
-           { withButtons && <Button className='redButton' onHandleClick={handleButtonClick} symbol={'x'} /> }
+        <div key={item.id} className='tag__item'>
+          <Tag key={item.id} id={item.id} tagName={item.displayName} selected={item.id === selectedTagId} onClick={handleTagClick} />
```
Comment (nikhilaravi) [inline src/components/linked-tags-list/index.js]: same here - split the Tag props onto new lines to make it more readable

File: src/components/tag-list/index.js [hunk 3]
```diff
@@ -57,12 +58,16 @@ class TagList extends Component {
...
-      pagination
+      pagination,
+      symbol,
+      handleButtonClick,
+      withButtons
     } = this.props;
     const list = items.map(item => {
       return (
         <div key={item.id} className='tag__item'>
           <Tag key={item.id} id={item.id} displayName={item.displayName} selected={item.id === selectedTagId} onClick={onTagClick} />
```
Comment (nikhilaravi) [inline src/components/tag-list/index.js]: same again here

File: src/components/view-pane/index.js [hunk 4]
```diff
@@ -80,36 +87,89 @@ class ViewPane extends Component {
         <div className='keyValueContainer'>
           <h4 className='keyValuePair'>Add new key value pair:</h4>
           <div className='keyValueInput'>
+          <Row>
             <Col xs={6}>
               <Input type='text' onChange={onChange} className='form-control' placeholder='add new key' />
             </Col>
             <Col xs={6}>
               <Input type='text' className='form-control addNewValue' placeholder='add value' buttonAfter={addButton}/>
```
Comment (nikhilaravi) [inline src/components/view-pane/index.js]: maybe here too..

**Comment by nikhilaravi** [comment]
cool! Nice :bookmark_tabs:

**Pull request merged**